### PR TITLE
added red alert on wrong choice

### PR DESCRIPTION
**Pull request by IlyaGEdgify**
added red alert on wrong choice
spec -
https://docs.google.com/document/d/1impR-USgCuvfbv384EIZJvgA8TGhfBlva4TGFNO4LZI/edit#heading=h.bimja1lti5ds

### CODE REVIEW

File: src/PosUi/alertsCounter/actions.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    state.prediction.predictions[0]?.json.predictionsList?.[0]?.dataList;
+  const firstLabel = firstGuess?.[0];
+  const accuracy = firstGuess?.[1];
+  const payload =
+    ![OTHER_FRUIT_LABEL, NO_FRUIT_LABEL, BAG_COVER, gt].includes(firstLabel) &&
+    accuracy > alertThreshold;
+  dispatch({
+    type: ALERTS_INCREASE,
+    payload,
+  });
```
Comment (sergeyedgify) [inline src/PosUi/alertsCounter/actions.js:27]: Why do you not dispatch action when the condition is true only? Payload should be 0 or 1 not boolean. For better code readability reason.

File: src/PosUi/alertsCounter/AlertsCounter.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  return (
+    // eslint-disable-next-line jsx-a11y/no-static-element-interactions
+    <div
+      className={`${classname} AlertsCounter`}
+      onClick={() => dispatch(resetAlerts())}
+    >
+      <div className="number">{value}</div>
+    </div>
+  );
+};
```
Comment (sergeyedgify) [inline src/PosUi/alertsCounter/AlertsCounter.jsx:23]: What do you think about to split it to dumb component and smart component (will have a connection to redux, and make a decision about value)?

Comment (IlyaGEdgify) [reply src/PosUi/alertsCounter/AlertsCounter.jsx:23]: There is no need in smarts components anymore - If needed I split the components to hooks (where I handle all logic and get things from store) and dumb component that get all data from hooks and represent it.

This component is to small and there is no need to add hooks only for 1 line

File: src/PosUi/alertsCounter/actions.js (hunk 1 above)
Comment (IlyaGEdgify) [reply src/PosUi/alertsCounter/actions.js]: fixed - dispatch only if needed

**Pull request merged**